src/NosCore.GameObject/Providers/ItemProvider/Item/WearableInstance.cs

```diff
@@ -146,5 +149,111 @@ public void SetRarityPoint()
                     break;
             }
         }
+
+        public void Sum(ClientSession clientSession, WearableInstance item)
+        {
+            if (!clientSession.HasCurrentMapInstance)
+            {
+                return;
+            }
+
+            if (Upgrade + item.Upgrade > 5)
+            {
+                return;
+            }
+
+            if ((
+                    (Item.EquipmentSlot != EquipmentType.Gloves) ||
+                    (item.Item.EquipmentSlot != EquipmentType.Gloves)) && (
+                    (Item.EquipmentSlot != EquipmentType.Boots) ||
+                    (item.Item.EquipmentSlot != EquipmentType.Boots)))
+            {
+                return;
+            }
+
```

Comment:
Move this to a helper so we can extract it easily when moving algorithms to a nugget package

Reply:
Maybe upgrade / update chances could be configurable though a config file, I don't think changing raw values should require a rebuild 

Also this would make it more clearer, this system currently looks like a copy and paste from OpenNos